.templates/docs.sh.template

```diff
@@ -0,0 +1,66 @@
+{% extends "base_script.sh.template" %}
```

Comment:
I think it makes sense to put the `fpga_config` stuff in the `docs` script instead of the `remote-script` since it's related to this content and not the remote wrapper. In that case, once `nengo-bones` is updated we can just extend the `docs` script with the `fpga_config` part and we can use the default `nengo-bones` version of `remote-script` which should be nice and clean.

Reply:
Oh... this `docs` template is here only because it uses the modified `base_script` template. When the slurm changes are merged into NengoBones, this file is not needed, we can use the default `docs` template.

Reply:
I feel like we should leave the `docs` template alone (i.e., don't put any modifiers in it) so that people can still run it on their local machines with their own configs. Putting the `fpga_config` stuff in the `remote-script` allows us to do that.

Reply:
Yea I understand that, hence "once nengo-bones is updated...". This is more a comment on the fact that I (subjectively) think it makes more sense to have the extra `fpga_config` content in the `docs` template instead of the `remote-script` template since it pertains to building the docs, not running remotely. Either way once updates are merged into `nengo-bones` we will need one of either `docs` or `remote-script` to be custom and I think it makes more sense for that to be `docs`.

Reply:
O sorry your second comment hadn't loaded when I typed this... That makes sense to me for leaving it in `remote-script`.

Reply:
I have a different opinion. With the `fpga_config` stuff in `remote-script` (because it really is only for the remote CI setup, you can build the docs by running the CI scripts directly (and you don't have to remember the exact `sphinx-build` command to do so). You can even call `.ci/docs.sh install` to install all of the necessary packages (except for `pandoc`) you need to do the docs build. If we put the `fpga_config` changes in the `docs.sh` template, it breaks this functionality since it's likely that the `fpga_config` in use would be the root `nengo-fpga` directory, and not in the home directory.